### ISSUE TITLE
Request failed: GET /analyse/startup-failure failed: no response

### ISSUE THREAD

**Issue by dimaqq** [docker/for-mac#6038]
Opening new issue after #4986

I think I get `Request failed: GET /analyse/startup-failure failed: no response` on every Docker "upgrade and restart".

I kinda like my custom config, so I don't "reset to defaults".

Just hitting "Exit" seems to allow the upgrade to proceed.

(Today's upgrade was from 4.1.0 to 4.2.0; earlier upgrades were from 4.0.x to 4.1.0 at least)

**Comment by StefanScherer** [comment]
Thanks @dimaqq, yes it is fixed in 4.2.0, but it only will take effect in future updates. Looking forward to new releases where you don‘t have to go through this again. 🤞
Linking #5834 for reference.

**Comment by joseluisq** [comment]
Right after apply the update.

![image](https://user-images.githubusercontent.com/1700322/142381368-c3e38ba9-63cd-4d38-9617-3ea7536becd5.png)
